src/test/java/org/folio/marccat/resources/HeaderTypeTest.java

```diff
@@ -8,31 +11,27 @@
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import java.util.Map;
-
-import static io.restassured.RestAssured.given;
-
 @RunWith(SpringRunner.class)
 @ActiveProfiles("test")
 public class HeaderTypeTest extends TestBase {
 
- @Test
+  @Test
+
   public void getDocumentCountById() {
 
-    String url = getURI( "/marccat/header-types");
+    String url = getURI("/marccat/header-types");
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("code", "1")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("code", "1").param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
+  @Test
+  public void getAuthorityHeadingTypes() {
 
+    String url = getURI("/marccat/auth-header-types");
+    Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
+    given().param("code", "008").param("lang", "eng").headers(headers).when().get(url).then().statusCode(200);
```

Comment:
What about testing the response body? Sure, it covers the code but it gives me no confidence that the API is returning the right thing.